src/page-templates/content-type-page/content-type-data.ts

```diff
@@ -40,6 +40,12 @@ export const getContentTypePageData = async (
         );
     }
 
+    if (initialSearchContent && contentType === 'Event') {
+        initialSearchContent = initialSearchContent.filter(
+            item => item.end_time && new Date(item.end_time) > currentTime
```

Comment:
Do we have plans to migrate the realm search function into our backend?

Also, should we expand the search function with a time param (and potentially more params for other use cases) to apply filtering logic at the backend and further reduce UI processing?

Reply:
Also let me know if this filtering logic should happen elsewhere in case this is not the right place.

Reply:
Yes there are plans for this, in this ticket here https://jira.mongodb.org/secure/RapidBoard.jspa?rapidView=2027&projectKey=DEVHUB&view=detail&selectedIssue=DEVHUB-1738&quickFilter=16572. There are just some steps that need to be taken before actually migrating.

And so I think eventually once it's moved to the backend, we can just do the date compare on the backend and handle the filtering there, no need to pass the time as a parameter.